src/ITensorNetworks/chain_rules.jl

```diff
@@ -6,12 +6,13 @@ function ChainRulesCore.rrule(
   ::typeof(split_network),
   tn::Matrix{ITensor};
   projector_center=default_projector_center(tn),
+  rotation=false,
 )
   function pullback(dtn_split::Matrix{ITensor})
     dtn = map(t -> replaceprime(t, 1 => 0), dtn_split)
-    return (NoTangent(), dtn, NoTangent())
+    return (NoTangent(), dtn, NoTangent(), NoTangent())
```

Comment:
I would have thought you would only need two outputs from the pullback, since you don't need to specify the derivatives for keyword arguments (as far as I know derivatives with respect to keyword arguments is not supported).